### PR TITLE
Add Status:Staled label

### PR DESCRIPTION
**Pull request by refs**

### CODE REVIEW

**Review by micbar — APPROVED** [review]

File: .github/settings.yml [hunk 1]
```diff
@@ -165,6 +165,8 @@ labels:
     color: "#ffffff"
   - name: Status:Tested
     color: "#62db68"
+  - name: Status:Staled
```
Comment (phil-davis) [inline .github/settings.yml]:
```suggestion
  - name: Status:Stale
```

IMO "Staled" is not a real word.

Comment (refs) [reply .github/settings.yml]: @phil-davis uhm, confusion:

image

I thought about using a past tense.

Comment (phil-davis) [reply .github/settings.yml]: At least not in modern English

Comment (refs) [reply .github/settings.yml]: If it sounds funny for a native, then I'd go with your gut feeling! thanks!

**Review by phil-davis — APPROVED** [review]

**Pull request merged**